cypress/integration/a11y.spec.js

```diff
@@ -0,0 +1,187 @@
+describe('Check accessiblity', () => {
+    const TOTAL_ELIBIGILITY_QUESTIONS = 3;
+
+    function checkA11y() {
+        cy.injectAxe();
+        cy.checkA11y();
+    }
+
+    function acceptSiteCookies() {
+        cy.setCookie('cookies_preferences_set', 'true');
+        cy.setCookie(
+            'cookies_policy',
+            '{"essential":true,"settings":true,"usage":true,"campaigns":true}'
+        );
+    }
+
+    beforeEach(() => {
+        acceptSiteCookies();
+    });
+
+    afterEach(() => {
+        cy.on('fail', () => {
+            cy.screenshot();
+        });
+    });
+
+    context('Pre login pages', () => {
+        beforeEach(() => {
+            cy.visit('/');
+        });
+        it('Get your document legalised', () => {
+            checkA11y();
+        });
+
+        it('Choose a service', () => {
+            cy.get('.govuk-button--start').contains('Start now').click();
+            checkA11y();
+        });
+    });
+
+    describe('Post login', () => {
+        function uploadTestFile() {
+            cy.get('.multi-file-upload__input').attachFile('test.pdf');
+            cy.get('.govuk-button--secondary').contains('Upload').click();
+            cy.wait(500);
+        }
+
+        function clickContinueBtn() {
+            cy.get('.column-two-thirds .govuk-button')
+                .contains('Continue')
+                .click();
+        }
+
+        function checkFirstRadioAndClickContinue() {
+            cy.get('[type="radio"].govuk-radios__input').first().check();
+            clickContinueBtn();
```

Comment:
I know that these are just a11y tests, but I have no idea what the first radio is, and what it does, where it goes etc. 

With testing library, it would look like this 
```suggestion
            const { getByLabelText, getByRole } = cy;
            getByLabelText('First Item').click()
            getByRole('button').click()
```
And then I would stop nesting it inside a function, since it is just as readable as "checkFirstRadioAndClickContinue" (if not more), and is more flexible.